l10n_es_pos_simplified_invoice/static/src/js/screens.js

```diff
@@ -0,0 +1,456 @@
+function l10n_es_pos_simplified_invoice_screens(instance, module){ //module is instance.point_of_sale
+    var QWeb = instance.web.qweb,
+    _t = instance.web._t;
+
+    var round_pr = instance.web.round_precision
+
+    module.ReceiptScreenWidget = module.ScreenWidget.extend({
+        template: 'ReceiptScreenWidget',
+
+        show_numpad:     false,
+        show_leftpane:   false,
+
+        show: function(){
```

Comment:
Acabo de usar include, sin tocar nada y ya me da error "is not a function". De todas formas he dejado solo lo que quiero modificar y está igual. No funciona.